examples/generalized_linear_models/GLM-truncated-censored-regression.ipynb

```diff
@@ -0,0 +1,1356 @@
+{
```

Comment:
<p>"it should be fairly intuitive to see that we will likely overestimate the slope.": but the simple linear regression below <em>underestimates</em> the slope, right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>fixed</p>

Reply:
<p>I still see "overestimate" :D</p><p> </p>

Reply:
<p>Oh, I mean in that I've fixed it in my local files. I've not committed it yet. Didn't know if it's the done thing to do loads of small commits, or store them up. </p>

Reply:
<p>should now be fixed</p>